includes/Modules/AdSense.php

```diff
@@ -188,7 +188,7 @@ protected function output_adsense_script() {
 			add_action(
 				'wp_body_open',
 				function() use ( $client_id ) {
-					if ( $this->adsense_tag_printed ) {
+					if ( $this->adsense_tag_printed || ! is_singular( 'amp_story' ) ) {
```

Comment:
Since this is used as exit condition here, it needs to be inverted (bail if `is_singular( 'amp_story' )`). Also, I think we should move this to a separate exit condition if clause between lines 187 and 188 - we don't need to add the hook then.